pkg/apis/core/v1alpha1/federatedtypeconfig_types.go

```diff
@@ -27,15 +27,16 @@ import (
 
 // FederatedTypeConfigSpec defines the desired state of FederatedTypeConfig
 type FederatedTypeConfigSpec struct {
-	// The configuration of the target type.  Kind will be set to the
-	// name of this resource regardles of the value provided.
+	// The configuration of the target resource.  Kind will be set to the
```

Comment:
I think 'resource' is an instance of an api type, and that a `FederatedTypeConfig` is configured with api types rather than a resource.  I think this distinction is important in the context of federation to know what level of abstraction is being discussed.

Reply:
People tend to use the term 'resource' to apply both to the type and to instances of that type. If you think 'type' is clearer here, I'm fine with that.